jobs/validate.yaml

```diff
@@ -67,9 +67,6 @@
       - jammy:
           hour: '0' # Midnight
           dow: '1,3,5'  # MWF
```

Comment:
Should we add jobs for `noble`? I know these should be added once we have charms that support `noble`, right?

Reply:
Not yet, not all the k8s charms will yet support noble.  Only some of them.  All we can test with is jammy until we migrate all of them.